Packs/Pulsedive/Integrations/Pulsedive/Pulsedive.py

```diff
@@ -380,21 +377,17 @@ def scan_result_command(client: Client, args: Dict[str, Any], api_key) -> List[C
             if 'data' in qid_data and qid_data['data']:
                 qid_data.update({'qid': qid, 'indicator': qid_data['data']['indicator']})
                 if qid_data['data']['type'] == 'url' or qid_data['data']['type'] == 'domain':
-                    try:
-                        screenshot = requests.get(
-                            qid_data['data']['properties']['dom']['screenshot']
-                        )
+                    if 'dom' in qid_data['data']['properties']:
```

Comment:
Is the fact that `'dom'` is not found in `qid_data['data']['properties']` the only reason for the `try except`?
Maybe you should leave the `try except` and insert the `if else` inside the `try`?

Reply:
Yes, your change works for me.